src/requirements/data/colleges/he.ts

```diff
@@ -1,15 +1,26 @@
 import { Course, CollegeOrMajorRequirement } from '../../types';
+import { courseIsAllEligible } from '../checkers-common';
 
 const humanEcologyRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Total Academic Credits',
+    description: '120 academic credits are required'
+      + 'PE courses and courses numbered 1000-1099 do not count towards the 120 credits',
+    source: 'http://courses.cornell.edu/content.php?catoid=41&navoid=11600#Cornell_Credit_Requirements',
+    checker: courseIsAllEligible,
+    operator: 'or',
+    fulfilledBy: 'credits',
+    minCount: 120,
+    progressBar: true
+  },
   {
     name: 'Human Ecology Credits',
     description: 'Students must complete a minimum of 43 Human Ecology credits from College Distribution, Major Requirements and electives.',
     source: 'https://www.human.cornell.edu/academics/policies/requirements',
     checker: (course: Course): boolean => course.acadGroup.includes('HE'),
     operator: 'or',
     fulfilledBy: 'credits',
-    minCount: 43,
-    progressBar: true
```

Comment:
Can you explain the discrepancy on progressBar? Some of these colleges have it set to true and others do not.

Reply:
Also - do we want the 120 credits to be the progress bar? I kinda feel like I never count my credits so a bar of it is less helpful than all the requirements in general.

Reply:
I think the only one that doesn't have progressBar set to true is Engineering (because there's no hard 120 credit limit)
Yea I think displaying # of requirements rather than credits would be more helpful! I'll just double check with Yuxuan and then update that!